lib/dom-event-delegate.js

```diff
@@ -50,6 +60,12 @@ Box.DOMEventDelegate = (function() {
 		// Without checking the existence of a parentNode and returning null, we would throw errors
 		while (!found && element && !isModuleElement(element)) {
 			element = element.parentNode;
+
+			// Stop if we hit a non-basic HTML element such as the DocumentNode
+			if (!isBasicElement(element)) {
+				break;
```

Comment:
Might be simpler to just add `&& element.parentNode` in the `while` condition since you really only care about hitting the top of the tree.